x/airdrop/keeper/airdrop_account.go

```diff
@@ -55,6 +57,27 @@ func (k Keeper) GetAllAirdropAccounts(
 	return
 }
 
+func (k Keeper) CreateAirdropAccount(
+	ctx sdk.Context,
+	airdropAccount *types.AirdropAccount,
+) (err error) {
+	if airdropAccount.OriginAccountCreated {
+		return fmt.Errorf("origin account already created")
```

Comment:
Let's add this to the error types
```suggestion
		return types.ErrOriginAccountExists
```

Reply:
https://github.com/ojo-network/ojo/pull/175/commits/f964f4920151d6cd588cf30fe48a849bae3e6543